### DISCUSSION TITLE
Can you use rest API to send logs from azure/aws

### DISCUSSION THREAD

**Discussion by thegreekman** [Security-Onion-Solutions/securityonion#5686] [Unsupported Versions] [answered]
I was wondering if there were any API calls to send logs from azure/aws devices to security onion.

**Comment by clairmont32** [comment]
What type of logs? You may already have support for sending logs through filebeats.

**Reply by thegreekman** [reply]
Any logs really, syslog for example. Just want to be able to send syslog from devices(Windows/Linux) in azure to SO.

**Reply by clairmont32** [reply]
You should be able to send that via filebeats/winfilebeats. Typically the winfilebeats only requires minimal changes to the config file to get them sent over _after_ you do a `so-allow` for the traffic source IP. 

https://docs.securityonion.net/en/latest/beats.html?highlight=filebeats

**Comment by weslambert** [comment, marked as answer]
Another option would be to use FIlebeat modules:

https://docs.securityonion.net/en/latest/filebeat.html#modules

This would allow you to pull in various logs from Azure/AWS:

https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-module-azure.html
https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-module-aws.html
